hadoop-hdds/server-scm/src/test/java/org/apache/hadoop/ozone/container/common/TestEndPoint.java

```diff
@@ -137,6 +137,8 @@ public void testGetVersion() throws Exception {
       assertNotNull(responseProto);
       assertEquals(VersionInfo.DESCRIPTION_KEY, responseProto.getKeys(0).getKey());
       assertEquals(VersionInfo.getLatestVersion().getDescription(), responseProto.getKeys(0).getValue());
+    } finally {
+      FileUtils.deleteDirectory(tempDir);
```

Comment:
I think JUnit should clean up its `@TempDir`.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-built-in-extensions-TempDirectory